test/evp_extra_test.c

```diff
@@ -1119,6 +1120,13 @@ static int test_EVP_MD_fetch(int tst)
         if (!TEST_ptr(ctx))
             goto err;
 
+        if (tst == 2)
+            setenv("OPENSSL_CONF", "test/default.cnf", 1);
+        if (tst == 3)
+            setenv("OPENSSL_CONF", "test/default-and-fips.cnf", 1);
+        if (tst == 4)
+            setenv("OPENSSL_CONF", "fips.cnf", 1);
```

Comment:
It's possible this might not work the way you think on certain platforms. I will have to check...

Reply:
Even if it does is this really what we want to do in the test?

Reply:
Oh, you did start a discussion on that...  sorry, that blew right past me for some reason.